README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# hw01
+## Jimin Han 101
+My name is **Jimin Han**. \
+I am from Korea. *Yes, South not North*.\
+My hobby is photography. 
+[Here](https://www.instagram.com/jiminhan_photography/) are my photos.  
+I hope to become proficient in R after taking this class. 
+
+At first there were some difficulties in doing this assignment: 
+1. Forking the HW (Easier than I thought). 
+2. Getting the correct directories. 
+3. Familiarizing myself with how computers work 
+4. My branch was ahead of my master file, so I had to redo. 
+5. I hope in the course of few weeks I'll be able to breeze through these 'minor' inconveniences. 
+6.The biggest part was commiting the images. I thought I had done everything right, but as a matter of fact I had not commited the image file itself. That took quite a long time to fix. 
```

Comment:
You can just leave everything numbered as 1. When you render the Markdown document each item is automatically numbered.

```
1. Forking the HW (Easier than I thought). 
1. Getting the correct directories. 
1. Familiarizing myself with how computers work 
1. My branch was ahead of my master file, so I had to redo. 
1. I hope in the course of few weeks I'll be able to breeze through these 'minor' inconveniences. 
1. The biggest part was commiting the images. I thought I had done everything right, but as a matter of fact I had not commited the image file itself. That took quite a long time to fix. 
```

1. Forking the HW (Easier than I thought). 
1. Getting the correct directories. 
1. Familiarizing myself with how computers work 
1. My branch was ahead of my master file, so I had to redo. 
1. I hope in the course of few weeks I'll be able to breeze through these 'minor' inconveniences. 
1. The biggest part was commiting the images. I thought I had done everything right, but as a matter of fact I had not commited the image file itself. That took quite a long time to fix.